.github/workflows/edocs-preview.yaml

```
@@ -0,0 +1,18 @@
+name: Create EPDocs preview
```

For the same reason of the previous comment, change to:

```suggestion
name: EPDocs - Static Content Web Preview
```